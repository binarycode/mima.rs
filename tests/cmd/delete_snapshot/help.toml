args = 'help delete-snapshot'
status.code = 0
