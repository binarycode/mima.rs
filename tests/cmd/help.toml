args = 'help'
status.code = 0
fs.sandbox = true
