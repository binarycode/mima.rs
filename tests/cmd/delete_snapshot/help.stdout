Delete snapshot

Usage: mima delete-snapshot <GUEST_ID> <SNAPSHOT_ID>

Arguments:
  <GUEST_ID>     Guest ID
  <SNAPSHOT_ID>  Snapshot ID

Options:
  -h, --help  Print help
